docx/document.py

```diff
@@ -205,6 +216,11 @@ def __init__(self, body_elm, parent):
         super(_Body, self).__init__(body_elm, parent)
         self._body = body_elm
 
+    @lazyproperty
+    def bookmarks(self):
+        """|Bookmarks| object providing access to main story bookmarks."""
+        raise NotImplementedError
```

Comment:
When needed for mocking purposes, a new method/property is added as a "stub". It nevertheless always has a complete docstring (that is formatted according to PEP 257). Its only line is `raise NotImplementedError` which causes the acceptance test to fail at just the right spot when it's next in line for implementation.